frontend/test/metabase/scenarios/joins/joins.cy.spec.js

```diff
@@ -188,7 +188,7 @@ describe("scenarios > question > joined questions", () => {
     cy.icon("add_data").click();
 
     enterCustomColumnDetails({
-      formula: "[Question 5 → Sum of Rating] / [Sum of Rating]",
+      formula: "[Question 5 → Sum of Rating] / [Sum of Total]",
```

Comment:
If you look at the field this tests uses, it is actually `Sum of Total`. Not sure how it passed originally.